php/kuiper/composer.json

```
@@ -0,0 +1,26 @@
+{
+  "require": {
+    "php": "^8.1",
+    "ext-json": "*",
+    "kuiper/web": "^0.8"
```

to align constraint on php based frameworks

```suggestion
    "kuiper/web": "~0.8"
```

---

kuiper framework is not released currently. ^0.8 equal to >=0.8 and < 0.9, ~0.8 equal to >= 0.8 and < 1.0，previous one is prefered

---

you are right, `composer` act differently for versions behind 1.0. maybe using `~0.8.0` will be clearer then